src/common/components/root/root.tsx

```
@@ -39,7 +39,8 @@ import { Avatar, AvatarFallback } from "../ui/avatar";
 import DebugModal from "./debug-modal";
 import { useGame } from "@/core/store/game-store";
 import { api } from "@/api/api";
-import { GameMap, Map } from "../map/new-game-map";
+import { GameMap, Map } from "../map/game-map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)